discord/appinfo.py

```diff
@@ -47,7 +47,7 @@
 
 
 class AppInfo:
-    """Represents the application info for the bot provided by Discord.
+    """Represents the application info for the bot from discord.
```

Comment:
```suggestion
    """Represents the application info for the bot provided by Discord.
```I prefer the original here